### PR TITLE
demo: modify the release image without creating a new container image

### PR DESCRIPTION
**Pull request by guillaumerose**
This exports all manifests to the VM disk, modify them, and then mount
them in the CVO pod.

---

This is an alternative to a previous experiment where I needed to upload a new release image.
This attempt is far less simple! 

/hold

### CODE REVIEW

File: snc.sh [hunk 1]
```diff
@@ -206,6 +206,17 @@ retry ${OC} replace -f pull-secret.yaml
 # Remove the Cluster ID with a empty string.
 retry ${OC} patch clusterversion version -p '{"spec":{"clusterID":""}}' --type merge
 
+# SCP the kubeconfig file to VM
+${SCP} $1/auth/kubeconfig core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/home/core/
```
Comment (praveenkumar) [inline snc.sh:210]: CI is failing because the `SCP` is not defined in the snc.sh side.

Comment (guillaumerose) [reply snc.sh:211]: thanks! done

File: snc.sh [hunk 2]
```diff
@@ -206,6 +207,17 @@ retry ${OC} replace -f pull-secret.yaml
...
 retry ${OC} patch clusterversion version -p '{"spec":{"clusterID":""}}' --type merge
 
+# SCP the kubeconfig file to VM
+${SCP} ${KUBECONFIG} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/home/core/
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo mv /home/core/kubeconfig /opt/'
+
+# Export all manifests to the disk, modify them and use them in the CVO
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} sudo mkdir /opt/release-manifests/
+CVO_POD_NAME=$(${OC} -n openshift-cluster-version get pods -o=name)
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} sudo KUBECONFIG=/opt/kubeconfig oc rsync -n openshift-cluster-version ${CVO_POD_NAME}:/release-manifests/ /opt/release-manifests/
```
Comment (praveenkumar) [inline snc.sh:217]: @guillaumerose you forget here `--` after SSH should be something like
```
${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo KUBECONFIG=/opt/kubeconfig oc rsync -n openshift-cluster-version ${CVO_POD_NAME}:/release-manifests/ /opt/release-manifests/'
```

**Comment by guillaumerose** [comment]
```
 + ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc core@api.crc.testing -- sudo KUBECONFIG=/opt/kubeconfig oc rsync -n openshift-cluster-version pod/cluster-version-operator-8668d7dbd8-wgrg9:/release-manifests/ /opt/release-manifests/
Warning: Permanently added 'api.crc.testing,192.168.126.11' (ECDSA) to the list of known hosts.
WARNING: cannot use rsync: rsync not available in container
WARNING: cannot use tar: tar not available in container
error: No available strategies to copy. 
```

Fails in the CI but works for me with 4.7.0-rc3 bundle :thinking:

**Comment by praveenkumar** [comment]
> ```
>  + ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc core@api.crc.testing -- sudo KUBECONFIG=/opt/kubeconfig oc rsync -n openshift-cluster-version pod/cluster-version-operator-8668d7dbd8-wgrg9:/release-manifests/ /opt/release-manifests/
> Warning: Permanently added 'api.crc.testing,192.168.126.11' (ECDSA) to the list of known hosts.
> WARNING: cannot use rsync: rsync not available in container
> WARNING: cannot use tar: tar not available in container
> error: No available strategies to copy. 
> ```
> 
> Fails in the CI but works for me with 4.7.0-rc3 bundle

@guillaumerose you do need to use `''` on the ssh command like I said https://github.com/code-ready/snc/pull/340#discussion_r580945383 here.

**Comment by guillaumerose** [comment]
Closed in favor of https://github.com/code-ready/snc/pull/342

**Pull request closed without merging**